install/amazon_rds/_03_setup_iam_policy_cli.mdx

```diff
@@ -0,0 +1,12 @@
+The following command creates a new IAM policy named pganalyze:
+
+```bash
+# First, download the policy JSON file
+curl -O https://pganalyze.com/docs/install/amazon_rds/pganalyze-collector-policy.json
```

Comment:
TODO: this link doesn't work. With ECS, we also have a long JSON, and we say something like:

> Now, save the following ECS task definition to a file, for example pganalyze_task.json:

We can do the same technically here, though I thought this is a bit more user friendly.
Well, if it's harder to try serving this json file under /docs (I wasn't able to find easy way to do so so far, sadly), I guess we should do the same as what ECS does.

Reply:
I'm not sure how to serve files under docs. My guess is we'd need to write a Gatsby GraphQL query that will run as part of the build, but looking around briefly, I don't see how to accomplish this. You may have better luck.

Alternately, we could do something like [this](https://github.com/gatsbyjs/gatsby/issues/872#issuecomment-298046002) and just copy the files to the /static/ folder we already have post-build. Thoughts?

Reply:
Thanks for the comment. Yeah I think there is some way to accomplish this, but sounds like it's gonna be complicated so I'll go as the same way as ECS (aka show policy json inline) for now.